frontend/src/metabase/containers/AddToDashSelectDashModal.tsx

```diff
@@ -32,18 +33,26 @@ interface AddToDashSelectDashModalProps {
 
 type DashboardPickerProps = ComponentPropsWithoutRef<typeof DashboardPicker>;
 
-class AddToDashSelectDashModal extends Component<
-  AddToDashSelectDashModalProps,
-  AddToDashSelectDashModalState
-> {
-  state = {
-    shouldCreateDashboard: false,
-  };
+export const AddToDashSelectDashModal = ({
+  card,
+  dashboards,
+  onClose,
+  onChangeLocation,
+}: AddToDashSelectDashModalProps) => {
+  const [shouldCreateDashboard, setShouldCreateDashboard] = useState(false);
 
-  navigateToDashboard: Required<CreateDashboardFormOwnProps>["onCreate"] =
-    dashboard => {
-      const { card, onChangeLocation } = this.props;
+  const mostRecentDashboardQuery = useMostRecentlyViewedDashboard();
 
+  const collectionId = mostRecentDashboardQuery.data?.collection_id;
+  // when no collectionId and loading is completed, we show root collection
+  // as user didnt' visit any dashboard last 24hrs
+  const collectionQuery = useCollectionQuery({
+    id: collectionId || "root",
```

Comment:
root is used only when `collection_id` is null

Reply:
`collection_id != null`

Reply:
I'm sorry, seems I didn't get the comment. Did you mean to use `collection_id != null` for `enabled`?

Reply:
@ranquild I hope I fixed it, but I didn't get the initial comment